app/models/user.rb

```diff
@@ -261,4 +261,6 @@ def introducer_dismissed?(slug)
   def send_devise_notification(notification, *args)
     devise_mailer.send(notification, self, *args).deliver_later
   end
+
+  def profile_unlocked? = reputation >= User::Profile::MIN_REPUTATION
```

Comment:
```suggestion
  def may_create_profile? = reputation >= User::Profile::MIN_REPUTATION
```

When I read "unlocked" I initially interpreted it as the profile having been made, so I think renaming it like this is wiser.